.travis.yml

```diff
@@ -2,7 +2,6 @@ language: ruby
 before_install:
 - gem install bundler -v '~> 1.5'
 rvm:
-- 2.1.10
```

Comment:
why did you remove this? 
I think it is a good idea to test on `2.1.10`. WDYT?

Reply:
because there is no support for this version

Reply:
this version was discontinued